client/src/components/CommentApp/components/CommentReply/index.tsx

```diff
@@ -187,19 +187,19 @@ export default class CommentReplyComponent extends React.Component<CommentReplyP
     return (
       <>
         <CommentHeader commentReply={reply} store={store} focused={isFocused} />
-        <p className="comment-reply__text">{reply.text}</p>
-        <div className="comment-reply__confirm-delete">
+        <p className="comment__text">{reply.text}</p>
+        <div className="comment__confirm-delete">
           {gettext('Are you sure?')}
           <button
             type="button"
-            className="comment-reply__button"
+            className="comment__button button button-small"
             onClick={onClickCancel}
           >
             {gettext('Cancel')}
           </button>
           <button
             type="button"
-            className="comment-reply__button comment-reply__button--primary"
+            className="comment__button button button-small no"
```

Comment:
Here and above: voluntary UI change for consistency between comments and replies. Those smaller buttons look a bit neater.